people.json

```diff
@@ -870,5 +870,11 @@
       "LinkedIn": "https://www.linkedin.com/in/adam-harald-joergensen/"
     },
     "contributor": true
+  },
+  {
+    "nick": "pzabkiewi",
+    "coordinates": [51.7235, 19.5005],
+    "links": {},
+    "contributor": true
```

Comment:
Where? https://github.com/SerenityOS/serenity/commits/master?author=cherrypickermaster is empty.

Reply:
is it a user map or contributor map ?

Reply:
I've made contributor: false now

Reply:
> is it a user map or contributor map ?

A user map that has contributors highlighted.